packages/ocamlbuild/ocamlbuild.0.14.2/opam

```diff
@@ -36,4 +42,5 @@ url {
     "md5=2f407fadd57b073155a6aead887d9676"
     "sha512=f568bf10431a1f701e8bd7554dc662400a0d978411038bbad93d44dceab02874490a8a5886a9b44e017347e7949997f13f5c3752f74e1eb5e273d2beb19a75fd"
   ]
-}
\ No newline at end of file
+}
+patches: "ocamlbuild-0.14.2.patch" {os = "win32"}
```

Comment:
The patch is 16K - I think it'd be better to do this using extra-source and reference the commit from the sunset repository instead?
```suggestion
}
extra-source "ocamlbuild-0.14.2.patch" {
  src: "https://raw.githubusercontent.com/ocaml-opam/opam-repository-mingw/354a87b397856f2a70024c5c83fc5001074935b6/packages/ocamlbuild/ocamlbuild.0.14.2/files/ocamlbuild-0.14.2.patch"
  checksum: "sha256=a9b7e1829a3304e5a073d8ddea29d3d8272698e93b7e1ee659ae5e31e5cfb6b9"
}
patches: "ocamlbuild-0.14.2.patch" {os = "win32"}
```

Reply:
done in 469a6255cfd0ec96611ac93eedcdb218abd6a699